<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGpsGpsdSourceDialog</class>
 <widget class="QDialog" name="CGpsGpsdSourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>146</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New GPSd source</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>GPSd host</string>
     </property>
     <property name="buddy">
      <cstring>leHost</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leHost">
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>GPSd port</string>
     </property>
     <property name="buddy">
      <cstring>lePort</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lePort">
     <property name="inputMask">
      <string>00000; </string>
     </property>
     <property name="text">
      <string>2947</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/dialog-cancel.png</normaloff>:/icons/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAccept">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/dialog-apply.png</normaloff>:/icons/dialog-apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CGpsGpsdSourceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
